src/pytorch/__init__.py

```diff
@@ -1,5 +1,18 @@
 '''
 High-performance PyTorch operations for neural network potentials
 '''
+import os.path
+import torch
+import site
 
-from NNPOps.OptimizedTorchANI import OptimizedTorchANI
\ No newline at end of file
+# look for NNPOps/libNNPOpsPyTorch.so in all the paths returned by site.getsitepackages()
+for path in site.getsitepackages():
```

Comment:
We talked about using only the relative paths to find the library. `site.getsitepackages` is unnecessary, because we create the package in a specific way.

Reply:
At installation __init__.py is located at the same place as libNNPOpsPyTorch.so, so we can simply use:

```python
torch.ops.load_library(os.path.join(os.path.dirname(__file__), 'libNNPOpsPyTorch.so'))
```

However, this breaks testing, since tests are located along the source files and make test runs lines like:
https://github.com/openmm/NNPOps/blob/293d90826371a20e63fa3bb21880cfd10132198a/CMakeLists.txt#L60

I believe this loads ```src/pytorch/__init__.py```, but since ```libNNPOpsPyTorch.so``` lives at sitepackages (and the build folder) the load fails. 

I only see two solutions to this:
1. Take the test scripts to another place (a test directory at the root?)
2. Modify the CMakeLists.txt to copy the test scripts to the build directory and run them from there.

2 seems like the best compromise to me. What do you think?

Reply:
> 2 seems like the best compromise to me. What do you think?

I agree.  That's what OpenMM does.